libtiledbsoma/src/soma/managed_query.h

```diff
@@ -5,7 +5,7 @@
  *
  * The MIT License
  *
- * @copyright Copyright (c) 2022 TileDB, Inc.
+ * @copyright Copyright (c) 2023 TileDB, Inc.
```

Comment:
I don't mean to a pedant but if 2022 was already present I think we usuallt do 2022-2023.  Not a replacement.